<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PhilipsWizSettingsEntry</class>
 <widget class="QWidget" name="PhilipsWizSettingsEntry">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>327</width>
    <height>149</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string notr="true">Philips Wiz Settings Entry</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Philips Wiz Device</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="2" column="0">
       <widget class="QCheckBox" name="UseCoolWhiteCheckBox">
        <property name="text">
         <string>Use Cool White</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="UseWarmWhiteCheckBox">
        <property name="text">
         <string>Use Warm White</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="IPEdit"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="IPLabel">
        <property name="text">
         <string>IP:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="WhiteStrategyComboBox"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="WhiteStrategyLabel">
        <property name="text">
         <string>White Strategy:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
